onnx/backend/test/case/node/image_decoder.py

```diff
@@ -39,7 +38,16 @@ def generate_checkerboard(width, height, square_size):
     return image
 
 
-def generate_test_data(extension, pixel_format="RGB", h=40, w=40, tile_sz=5):
+def generate_test_data(
+    extension, frozen_data, pixel_format="RGB", h=32, w=32, tile_sz=5
+):
+    try:
+        # pylint: disable=import-outside-toplevel
+        import cv2
+    except ImportError:
+        # Since opencv-python is not installed to generate test data for the ImageDecoder operator
+        # directly use the frozen data from image_decoder_data.py.
+        return frozen_data.data, frozen_data.output
```

Comment:
Finally I decide to change the behavior as:
1. If the env has installed opencv-python, still use opencv-python to generate input/output.
2. If the env does not have installed opencv-python, use frozen data from image_decoder_data.py instead. I only comment this behavior in the code since it's more like implementation detail. By contrast, printing out the message will be quite redundant when running test generation.

I also verifies these 2 behaviors in the CIs.